docker/postgres/docker-entrypoint-initdb.d/mock-data/interfaith_DDL.sql

```diff
@@ -46,13 +54,22 @@ CREATE TABLE need (
   need_active BOOLEAN NOT NULL
 );
 
--- DROP TABLE IF EXISTS needcallout cascade;
--- CREATE TABLE needcallout (
---   needcallout_id BIGINT PRIMARY KEY NOT NULL AUTO_INCREMENT,
---   needcallout_quantity INT NOT NULL,
---   user_id BIGINT REFERENCES user,
---   callout_id BIGINT REFERENCES callout,
---   active BOOLEAN NOT NULL
--- );
+DROP TABLE IF EXISTS needcallouts cascade;
+CREATE TABLE needcallouts (
+  needcallout_id BIGSERIAL PRIMARY KEY,
+  needcallout_quantity INT NOT NULL,
+  need_id BIGINT REFERENCES needs,
+  callout_id BIGINT REFERENCES callouts,
+  needcallout_active BOOLEAN NOT NULL
+);
+
+DROP TABLE IF EXISTS donations cascade;
+CREATE TABLE donations (
+  donation_id BIGSERIAL PRIMARY KEY,
+  donation_quantity INT NOT NULL,
+  donation_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Is this the created date or last modified date? Either way, it is doing the correct thing so it's not a blocking issue